modules/programs/zsh/zsh-abbr.nix

```diff
@@ -0,0 +1,40 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.programs.zsh.zsh-abbr;
+in {
+  meta.maintainers = [ hm.maintainers.ilaumjd ];
+
+  options.programs.zsh.zsh-abbr = {
+    enable =
+      mkEnableOption "zsh-abbr - zsh manager for auto-expanding abbreviations";
+
+    abbrs = mkOption {
+      default = { };
+      example = {
+        l = "less";
+        gco = "git checkout";
+      };
+      description = ''
+        An attribute set that maps aliases (the top level attribute names
+        in this option) to abbreviations. Abbreviations are expanded with
+        the longer phrase after they are entered.
+      '';
+      type = types.attrsOf types.str;
```

Comment:
The type should be the first attribute in the option, i.e., before the `default = { };` line.